src/Tasks.UnitTests/ResolveNonMSBuildProjectOutput_Tests.cs

```diff
@@ -119,32 +128,32 @@ public void TestResolve()
 
             // non matching project in string
             projectOutputs = new Hashtable();
-            projectOutputs.Add("{11111111-1111-1111-1111-111111111111}", @"obj\wrong.dll");
+            projectOutputs.Add("{11111111-1111-1111-1111-111111111111}", Path.Combine("obj", "wrong.dll"));
```

Comment:
This seems like something that might have been failing for quite a while but went unnoticed because we don't test full framework/macOS. Is that possible? If so, do you think we should add that as one of the standard checks?

Reply:
.NET ("full framework") runs only on Windows. There is no "full framework/macOS" scenario.